### ISSUE TITLE
PR Request for ARIA in HTML

### ISSUE THREAD

**Issue by siusin** [w3c/transitions#380]
# Document title, URLs, estimated publication date
ARIA in HTML
https://www.w3.org/TR/html-aria/
W3C Proposed Recommendation 28 September 2021

# Abstract
This specification defines the authoring rules (author conformance requirements) for the use of Accessible Rich Internet Applications (WAI-ARIA) 1.1 and Digital Publishing WAI-ARIA Module 1.0 attributes on [HTML] elements. This specification's primary objective is to define requirements for use with conformance checking tools used by authors (i.e., web developers). These requirements will aid authors in their development of web content, including custom interfaces/widgets, that makes use of ARIA to complement or extend the features of the host language [HTML].

# Status
ARIA in HTML is an [HTML] specification module. Any HTML features, conformance requirements, or terms that this specification module makes reference to, but does not explicitly define, are defined in the HTML specification.

This document was published by the Web Applications Working Group as a Proposed Recommendation. This document is intended to become a W3C Recommendation.

GitHub Issues are preferred for discussion of this specification. Alternatively, you can send comments to our mailing list. Please send them to public-webapps@w3.org (subscribe, archives).
Publication as a Proposed Recommendation does not imply endorsement by the W3C Membership.
This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.
The W3C Membership and other interested parties are invited to review the document and send comments through 21 September 2021. Advisory Committee Representatives should consult their WBS questionnaires. Note that substantive technical comments were expected during the Candidate Recommendation review period that ended 21 September 2021.
This document was produced by a group operating under the W3C Patent Policy. W3C maintains a public list of any patent disclosures made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains Essential Claim(s) must disclose the information in accordance with section 6 of the W3C Patent Policy.
This document is governed by the 15 September 2020 W3C Process Document.

# Link to group's decision to request transition
https://github.com/w3c/html-aria/issues/348
(closed with 24 supports, no objections)

# Changes
Only editorial changes since CR was published.
https://github.com/w3c/html-aria/commits/gh-pages

# Requirements satisfied
None.

# Dependencies met (or not)
HTML Accessibility API Mappings 1.0 and SVG Accessibility API Mappings are in WD status. No changes to their status since CR but not a stopper for the transition.

# Wide Review
The horizontal review results were tracked in [#273](https://github.com/w3c/html-aria/issues/273), and there was not any outstanding issue.

# Issues addressed
No open issues for [1.0](https://github.com/w3c/html-aria/labels/To%20Do%20for%201.0).

# Formal Objections
None.

# Implementation
An [implementation report](https://w3c.github.io/html-aria/results/implementation-results.html) is available, documenting at least two independent implementations for every feature.

# Patent disclosures
https://www.w3.org/groups/wg/webapps/ipr

**Comment by plehegar** [comment]
For the record: "A Proposed Recommendation may identify itself as intending to allow new features (class 4 changes) after its initial publication as a Recommendation,"

I don't see any of that in the proposed SOTD. Is it intentional?

**Comment by siusin** [comment]
No, we will not add new features to this version after REC. We will start working on a new version of ARIA in HTML.

**Comment by plehegar** [comment]
(note: we ran out of time today to look at this transition request. we'll look at it on Tuesday next week)

**Comment by swickr** [comment]
Transition approved.

**Comment by plehegar** [comment]
https://www.w3.org/TR/2021/PR-html-aria-20210930/
